src/main/resources/db/migration/R__TypeCodes.sql

```diff
@@ -440,6 +440,13 @@ VALUES (1, 'Achieved'),
        (3, 'Unlikely')
 ON CONFLICT (id) DO UPDATE SET name = excluded.name;
 
+INSERT INTO accelerator.report_quarters (id, name)
+VALUES (1, 'Q1'),
+       (2, 'Q2'),
+       (3, 'Q3'),
+       (4, 'Q4')
+ON CONFLICT (id) DO UPDATE SET name = excluded.name;
```

Comment:
My first reaction to this was, "Can't we use an integer that's constrained from 1 to 4?" but on reflection, I think this is a nice bit of added type safety.